src/components/SortOptions.jsx

```diff
@@ -1,23 +1,22 @@
-const SortOptions = () => {
-
+const SortOptions = (props) => {
+    
+    const { setOrder, setSortBy } = props
+    
     return (
-        <div className="sort-topics">
+        <div className="sort-options">
 
             <label htmlFor="sort-select">Sort by:</label>
-            <select name="sort-select" id="sort-select">
-                <option value="date">Date</option>
-                <option value="comment-count">Comment Count</option>
+            <select name="sort-select" id="sort-select" onChange={(e) => setSortBy(e.target.value)}>
+                <option value="created_at">Date</option>
+                <option value="comment_count">Comment Count</option>
                 <option value="votes">Votes</option>
             </select>
 
-            
-            <label htmlFor="order"></label>
-
-            <input type="radio" id="desc" name="order" value="desc" checked></input>
-            <label htmlFor="desc">Desc</label>
-
-            <input type="radio" id="asc" name="order" value="asc"></input>
-            <label htmlFor="asc">Asc</label>
+            <label htmlFor="order-select">Order:</label>
```

Comment:
This is kinda picky (and very much optional)
As there are only two options for the order, you might want to make this a toggle-able button rather than a drop down to save the user some clicks 💭